src/test/mocks/fixtures/get-v1-employees-employee_id-jobs.json

```diff
@@ -7,7 +7,7 @@
     "payment_unit": "Year",
     "primary": true,
     "title": "Client Support Director",
-    "state_wc_covered": "null,",
+    "state_wc_covered": null,
```

Comment:
Fixes a Speakeasy validation that expects this field to be null but not a string.